.xo-config.js

```diff
@@ -50,7 +43,14 @@ module.exports = {
 				svg: 'always'
 			}
 		],
-		'unicorn/no-fn-reference-in-iterator': 'off'
+
+		// Temporarily disabled
+		'@typescript-eslint/no-invalid-void-type': 'off',
+		'@typescript-eslint/no-extra-non-null-assertion': 'off',
+		'@typescript-eslint/no-unnecessary-condition': 'off',
+		'@typescript-eslint/no-floating-promises': 'off',
```

Comment:
Will you allow `void` on this instead?

Reply:
Do you think it makes sense to explicitly annotate promise-returning functions with `void` if they don't need to be await'ed?

Reply:
It'd force this everywhere 😬

```diff
- features.add({
+ void features.add({
```

But I think it's worth it everywhere else. It's good that the error message also suggests that you can just prepend `void` to fix the issue.

Reply:
Done